src/Nethermind/Nethermind.Facade/BlockchainBridge.cs

```diff
@@ -74,6 +78,8 @@ public class BlockchainBridge : IBlockchainBridge
             _wallet = wallet ?? throw new ArgumentException(nameof(wallet));
             _transactionProcessor = transactionProcessor ?? throw new ArgumentException(nameof(transactionProcessor));
             _ecdsa = ecdsa ?? throw new ArgumentNullException(nameof(ecdsa));
+            _blockFinder = new BlockFinder(_blockTree);
+            _logFinder = new LogFinder(_blockFinder, _receiptStorage);
```

Comment:
nice too

Reply:
It might be good to inject them when we go for DI, but not needed for now.